projects/ui-components/package.json

```diff
@@ -20,10 +21,9 @@
     "@angular/platform-browser": "^14.2.12",
     "@angular/platform-browser-dynamic": "^14.2.12",
     "@angular/router": "^14.2.12",
-    "@angular-material-components/datetime-picker": "^8.0.0",
-    "@laserfiche/lf-api-client-core": "^1.1.2",
-    "@laserfiche/lf-js-utils": "^4.0.4",
-    "@laserfiche/lf-resource-library": "^4.0.3",
+    "@laserfiche/lf-api-client-core": "^1.1.3",
+    "@laserfiche/lf-js-utils": "^4.0.7",
+    "@laserfiche/lf-resource-library": "^5",
```

Comment:
make this ^5.0.0
I think 5 is "okay" but I've gotten weird things happen when using it before

Reply:
updated